### ISSUE TITLE
Some java.lang.Math fields give NaN when accessing reflectively

### ISSUE THREAD

**Issue by jburse** [leaningtech/cheerpj-appletrunner#24]
Dear Leaningtech,

I am about to complete my test suite. I stepped over an interestig little glitch. My code uses reflection to access some fields. This works most of the time, exception for some fields of java.lang.Math, where I get a NaN, but I should get something else. My code reads:

```
Object res= field.get(obj);
System.err.println("field="+field+", obj="+obj+", res="+res);
return res;
```

The expected result is:

`field=public static final double java.lang.Math.PI, obj=null, res=3.141592653589793`

What happens in CheerpJ is the following:

`field=public static final double java.lang.Math.PI, obj=null, res=NaN`

**Comment by jburse** [comment]
Interestingly even the debugger pops up, which I have no explanation for:

**Comment by alexp-sssup** [comment]
This problem should be fixed in CheerpJ Applet Runner 6.7
